boskos/ranch/ranch_test.go

```diff
@@ -228,7 +228,7 @@ func TestAcquireRoundRobin(t *testing.T) {
 		}
 	}
 
-	for idx, _ := range c.Resources {
```

Comment:
[Doesn't look like it](https://github.com/kubernetes/test-infra/blob/b59e249f7821090ade1d0949dc09a5d18fe17423/verify/verify-gofmt.sh), but feel free to keep the changes.